testing/batch_processing/test_batch_processing.py

```diff
@@ -46,7 +46,9 @@ def test_batch_processing_results(csv_filepath, row, column):
     # These values should be considered a failure, but because the cause is known, skipping the
     # test saves us the intermittent interruptions on PRs until we can fix the issue.
     # TODO: Remove this skip once we can implement a long-term workaround for the issue
-    if actual_value in [0.780605541771859, 0.77461183127549, 54.42425823974834]:
+    if (actual_value == pytest.approx(0.780605541771859) or
+            actual_value == pytest.approx(0.77461183127549) or
+            actual_value == pytest.approx(54.42425823974834)):
```

Comment:
> [This commit](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3771/commits/14eb91cbba05bde51a91bbd41e5d5ed077340b6e)  introduces a change in the computation of the s-form, and re-introduces the flakiness from #3597 into the batch processing test.

If I'm inferring correctly from this, it sounds like the tested failed once here due to the Platinum CPU. Or... did it? Because of the rebase, I can't see the failing CI...

I ask because I'm not sure I understand how the sform change re-introduced flakiness into the batch processing test. I'd love to see an example of how the test failed, and what tolerance the values differed by.

Reply:
I could have sworn I saw a failure at the "Validate new results" stage which showed numbers very close to the Platinum CPU ones (which, incidentally, are also the numbers my own laptop produces, although it has a different CPU)... in fact that test run passed when I triggered a re-run, whereas the failure you linked would have failed again.

Ah! But I did a force-push to this branch, and I think all the test runs from those previous commits are gone as a result.

Anyway, since my CPU produces very similar results to the Platinum CPU, I can confirm that this PR changes the numbers a little bit (but still within the floating point comparison threshold). Does that address your concern?